mopidy/mpd/actor.py

```diff
@@ -57,9 +60,7 @@ def on_stop(self):
         process.stop_actors_by_class(session.MpdSession)
 
```

Comment:
I think this looks good, and I like that fact that it will complain when we add a new core event and forgets to update MPD.

As for URI mapper maintenance, I'm OK with a if/elif/elif block after the above code.